terraform/digitalocean/k8s/data.tf

```diff
@@ -0,0 +1,3 @@
+data "digitalocean_kubernetes_versions" "k8s" {
+  version_prefix = "${var.version}."
+}
```

Comment:
No logic to create separate file for one `data` resource. Feel free move it to `main.tf` or rename file to `init.tf`

When we have it many - good idea to create `init.tf` where locate `backend`, `provider` and `data` definitions

Reply:
I move to main.tf